hack/vmimage-validate.sh

```diff
@@ -25,4 +25,4 @@ echo "Validating: ${IMAGE_RESOURCENAME}"
 go generate ./... && go run -ldflags "-X main.gitCommit=$(git rev-parse --short=10 HEAD)" ./cmd/vmimage -buildResourceGroup "${BUILD_RESOURCE_GROUP}" -imageResourceGroup "${IMAGE_RESOURCEGROUP}" -image "${IMAGE_RESOURCENAME}" -imageStorageAccount "${IMAGE_STORAGEACCOUNT}" -validate -preserveBuildResourceGroup
 
 # Currently there are only 3 logs we want to capture
-mv /tmp/{yum_update_info,yum_check_update,scap-report.html} ${ARTIFACTS:-/tmp}/ || true
+mv /tmp/{yum_updateinfo,yum_updateinfo_list_security,yum_check-update,scap-report.html} ${ARTIFACTS:-/tmp}/ || true
```

Comment:
For some reason we have renamed the old files and are mixing hyphens and underscores. Can we consistently use snake case here?

Reply:
We previously had `yum updateinfo` logging into `/tmp/yum_update_info` (why `_` in the middle of `updateinfo`?) and `yum check-update` logging into `/tmp/yum_check_update`.

I changed the names to match commands: `yum check-update` now writes to `/tmp/yum_check-update` and `yum check-update` to `/tmp/yum_check-update`. So basically replacing space with underscore and leaving rest of the command as is.

I don't mind replacing both hyphens and spaces with underscores, if that's what you mean.

Reply:
Yes, let's use underscores everywhere.